src/integrationTest/java/uk/nhs/hee/trainee/details/api/CctResourceIntegrationTest.java

```diff
@@ -113,6 +113,7 @@ void setUp() throws IOException {
             "endDate": "2025-01-01",
             "wte": 0.5
           },
+          "cctDate": "2024-01-01",
```

Comment:
This would have caused a number of tests to fail prior to relaxing the validation constraint